routes/routes.go

```diff
@@ -48,18 +62,3 @@ func notFound(w http.ResponseWriter, r *http.Request) {
 	w.WriteHeader(http.StatusNotFound)
 	json.NewEncoder(w).Encode(nil)
 }
-
-var routes = types.Routes{
```

Comment:
this means we have to explicitly add the test if we add another route

But that might actually be a good thing